ExampleMod/Items/Accessories/ExampleDoubleJump.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Items.Accessories
+{
+	/// <summary>
+	/// This is an example of a <see cref="ModDoubleJump"/> with inheritance to showcase increasing jumpheight and visuals, used in <see cref="ExampleDoubleJumpAccessory"/>.
+	/// A simpler ModDoubleJump is <see cref="SimpleDoubleJump"/>
+	/// </summary>
+	public abstract class ExampleDoubleJump : ModDoubleJump
+	{
+		// Optional field we use in our inheritance below, to not override the same hooks again just to adjust one number.
+		// It represents the jump height increase and the amount of dust we spawn
+		public abstract int Power { get; }
+
+		public override float Jump(ref bool playSound) {
+			CombatText.NewText(player.getRect(), Color.Lime, "'" + Power + "!'", true);
```

Comment:
What's this for?

Reply:
I think this is just to demonstrate to the modder that it is working

Reply:
Yeah it's for representing which of the three jumps is currently triggered ingame